### ISSUE TITLE
KmsKeyId not valid property for AWS::Logs::LogGroup

### ISSUE THREAD

**Issue by vennemp** [aws-cloudformation/cfn-python-lint#1763]
*cfn-lint version: (`cfn-lint --version`)*
0.39.0
*Description of issue.*
```yaml  
  CloudWatchKMS:
    Type: AWS::KMS::Key
    Properties:
      Enabled: true
      EnableKeyRotation: true
      KeyPolicy:
        Version: '2012-10-17'
        Statement:
          - Sid: Enable IAM User Permissions
            Effect: Allow
            Principal:
              AWS: !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:root
            Action: kms:*
            Resource: '*'
          - Sid: Allow Cloudwatch Access
            Effect: Allow
            Principal:
              Service: logs.region.amazonaws.com
            Action:
              - kms:Encrypt*
              - kms:Decrypt*
              - kms:ReEncrypt*
              - kms:GenerateDataKey*
              - kms:Describe*
            Resource: '*'
  MessagesLogCWGroup:
    Type: AWS::Logs::LogGroup
    Properties: 
      KmsKeyId: !GetAtt CloudWatchKMS.Arn
      LogGroupName: messages
      RetentionInDays: 30
```

E3002 Invalid Property Resources/MessagesLogCWGroup/Properties/KmsKeyId
template.yaml:627:7

**Comment by kddejong** [comment]
You can run `cfn-lint --update-specs` to get the update now or we will have the next release out this week.

**Comment by vennemp** [comment]
ok so i would run: 
```
cfn-lint --template template.yaml --regions us-gov-west-1 --update-specs
```

**Comment by kddejong** [comment]
Just run `cfn-lint --update-specs` and then run `cfn-lint --template template.yaml --regions us-gov-west-1`

**Comment by PatMyron** [comment]
Don't think [`AWS::Logs::LogGroup.KmsKeyId`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.html#cfn-logs-loggroup-kmskeyid) has been published to the [`us-gov-west-1`](https://s3.us-gov-west-1.amazonaws.com/cfn-resource-specifications-us-gov-west-1-prod/latest/CloudFormationResourceSpecification.json) [Resource Specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) yet

so another case of [Resource Specifications lagging recent releases](https://github.com/aws-cloudformation/cfn-python-lint/issues/1219), but should be present in the [`us-east-1`](https://d1uauaxba7bl26.cloudfront.net/latest/gzip/CloudFormationResourceSpecification.json) default now

**Comment by vennemp** [comment]
Thank you. I appreciate you following up.
